charms/kfp-profile-controller/src/components/pebble_components.py

```diff
@@ -50,7 +50,6 @@ def get_layer(self) -> Layer:
                         "command": "python /hooks/sync.py",  # Must be a string
                         "startup": "enabled",
                         "environment": {
-                            "minio-secret": inputs.MINIO_SECRET,
```

Comment:
I understand that you have tested with minio charm and profile-controller works as expected (since we have some different env variables from upstream)

Reply:
This was tested extensively. I believe the reason is that the lower case env variable is not used anywhere, so we are just keeping the upper case MINIO_SECRET and other similar vars.